docs/data/sources/google-tag-manager-client.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Amplitude Analytics Browser SDK
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
+---
+
+Client-side Google Tag Manager Template for Amplitude Analytics. The tag utilizes the [Amplitude Marketing Analytics SDK](../../data/sdks/marketing-analytics-browser/) for data collection.
```

Comment:
```suggestion
This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Amplitude Marketing Analytics SDK](../../data/sdks/marketing-analytics-browser/) for data collection.
```

Reply:
Added "This is" and replaced "utilize" to "use" for all pages.